Automattic/newspack-plugin
# Update setup wizard to use action cards for integrations step

### All Submissions:






### Changes proposed in this Pull Request:

This PR simplifies the Integrations interface by using `ActionCard`s with checkboxes.

### How to test the changes in this Pull Request:

1. Navigate to `/admin.php?page=newspack-setup-wizard#/integrations`
2. Switch to this branch
3. You should now see the `ActionCard`s
4. Disconnect then Connect Jetpack, GSK, Mailchimp to check if the "Connect" buttons are working properly

### Other information:
